scripts/check_urls/check_urls.py

```diff
@@ -223,24 +223,14 @@ def send_slack_message(invalid_uris_list: List[IOUrl]):
     urls_black_list = {
         # 403 when this check runs (in the middle of the night)
         "https://id.lepida.it/docs/manuale_utente.pdf",
-        # still not available
-        "https://io.italia.it/carta-giovani-nazionale/informativa-beneficiari",
-        # still not available
-        "https://io.italia.it/carta-giovani-nazionale/guida-beneficiari",
-        # the article is available but the response status code is 403
-        "https://help.mixpanel.com/hc/en-us/articles/115004494803-Disable-Geolocation-Collection",
-        "https://assets.cdn.io.italia.it",
-        "https://www.trusttechnologies.it/wp-content/uploads/SPIDPRIN.TT_.DPMU15000.03-Guida-Utente-al-servizio-TIM-ID.pdf",
-        "https://www.trusttechnologies.it/contatti/#form",
-        "https://support.namirial.com/it/faq/faq-tsp/faq-tsp-spid",
-        "https://paytipper.com/wp-content/uploads/2021/02/logo.png",
-        "https://fims-dev-app-provider.azurewebsites.net/",
-        # returns a 404 anytime the check runs but it actually works fine
+         # returns a 404 anytime the check runs but it actually works fine
```

Comment:
Why did we put here a spacer? 🤔

Reply:
No idea, removed in https://github.com/pagopa/io-app/pull/5662/commits/38026e31acf64f1f3b39a0d56c4db6b132144e01, thanks.